test/common/http/http1/codec_impl_test.cc

```diff
@@ -1155,7 +1155,7 @@ TEST_F(Http1ServerConnectionImplTest, IgnoreUpgradeH2cCloseEtc) {
   TestHeaderMapImpl expected_headers{{":authority", "www.somewhere.com"},
                                      {":path", "/"},
                                      {":method", "GET"},
-                                     {"connection", "Close,Etc"}};
```

Comment:
This change is due to the prod code change I pointed out above. Let me know what you think.

Reply:
We could summon @PiotrSikora ?